src/ui/components/Notice/styles.scss

```diff
@@ -0,0 +1,97 @@
+@import "~core/css/inc/mixins";
+@import "~core/css/inc/vars";
+@import "~ui/css/vars";
+@import "~photon-colors/colors";
+
+.Notice {
+  border-radius: $border-radius-s;
+  display: grid;
+  font-size: $font-size-s;
+  grid-gap: 4px;
```

Comment:
Most of our base units are `6px` multiples. Should this really be `4px`? If it totally looks right compared to mocks that's okay, but I'd assume 6 is preferred.

Reply:
The [spec](https://design.firefox.com/photon/components/message-bars.html) says it should be 4px.